src/graphql/typeDefs/dataTypes/session/mySession.graphql

```diff
@@ -3,7 +3,7 @@ type MySession @key(fields: "id") {
   id: ID!
   eventId: ID!
   slug: Slug!
-  speakers: [Profile]
+  speakers: [PublicProfile]
```

Comment:
shouldn't this be required?

Reply:
I guess, that is old code there

Reply:
@csell, this makes sense, but it should then be added to all the session types as well, only mySession

Reply:
yea prolly a good change to make everywhere when we run across it.